odoo/addons/base/models/res_users.py

```diff
@@ -248,8 +248,8 @@ def _get_company(self):
 
     # overridden inherited fields to bypass access rights, in case you have
     # access to the user but not its corresponding partner
-    name = fields.Char(related='partner_id.name', inherited=True)
-    email = fields.Char(related='partner_id.email', inherited=True)
+    name = fields.Char(related='partner_id.name', inherited=True, readonly=False)
+    email = fields.Char(related='partner_id.email', inherited=True, readonly=False)
```

Comment:
clash ?

Reply:
Nope. The ORM will not define those fields as inherited, because they are defined on the child model.